docs/REST API/rest-api.md

```diff
@@ -8298,6 +8315,13 @@ _Available in Fleet Premium_
         "host_batch_size": 0
       }
     },
+    "integrations": {
```

Comment:
@rachaelshaw @getvictor What API field does [this form field](https://www.figma.com/file/p81nWodxL04YD7iNyr9xVa/%2317230-Fleet-in-your-calendar?type=design&node-id=513%3A9054&mode=design&t=7Nyma5ovXM6NPVWj-1) translate to? Are we missing a field here?